content/docs/how-we-work/drupal/coding.md

```diff
@@ -23,16 +23,33 @@ If we have written a generic implementation, others in the project can use it ve
 
 ## Contribution first
 
-We have a strong enthusiasm for [open source contributions]({{< relref "../open-source-contribution/" >}}), particularly in coding. Whenever we encounter a problem within a module, our first instinct is to find a solution and submit a patch to address it. Similarly, if we come across someone seeking assistance in the drupal.org issue queue and we happen to know the answer, we gladly provide a comment to help them out. Additionally, when we develop a generic solution that can solve a common problem, we will submit it as a module. We believe in the power of collaboration and the impact of shared knowledge.
+We have a strong enthusiasm for [open source contributions]({{< relref "../open-source-contribution/" >}}), particularly in [Drupal]({{< relref "../open-source-contribution//drupal-contribution/" >}}) coding. Whenever we encounter a problem within a module, our first instinct is to find a solution and submit a patch to address it. Similarly, if we come across someone seeking assistance in the drupal.org issue queue and we happen to know the answer, we gladly provide a comment to help them out. Additionally, when we develop a generic solution that can solve a common problem, we will submit it as a module. We believe in the power of collaboration and the impact of shared knowledge.
 
 ## Security
 
 While writing code in Drupal, it is mandatory to keep our code secure and apply the best practices.
 
 - [Writing secure code for Drupal](https://www.drupal.org/docs/administering-a-drupal-site/security-in-drupal/writing-secure-code-for-drupal)
 
-## Coding Standards
+## Drupal-specific Coding Standards
 
-We apply the Drupal coding standards when coding in Drupal projects. We always make sure that these standards are up-to-date and version-independent.
+- We follow [Drupal Coding Standards](https://www.drupal.org/docs/develop/standards), which include PHP, CSS, JavaScript, Twig, YAML, and more.
 
-- [Drupal coding standards and best practices](https://www.drupal.org/docs/develop/standards)
+- We ensure [Code Quality with DCQ](https://www.drupal.org/project/dcq) which includes the Coder module to ensure code meets Drupal standards.
```

Comment:
Does someone use this module at Axelerant? I know some of our projects use https://github.com/axelerant/drupal-quality-checker

Reply:
I checked it, and it was last updated in March 2023, but it works with Drupal 7.x. I'll keep [this](https://github.com/axelerant/drupal-quality-checker) in mind.

Reply:
Yes. I think we can mention https://github.com/axelerant/drupal-quality-checker instead of this

Reply:
@hussainweb and @Kalaiselvan88: Thanks. As per your suggestion, I changed it to Drupal Quality Checker.